funcs_scripts.php

```diff
@@ -88,14 +108,30 @@ function delete_file_if_exists($filename)
 function module_is_recipe()
 {
     global $MODULE_DIR;
-    if (strpos('/recipe-', $MODULE_DIR) !== false
-        || strpos('/silverstripe-installer', $MODULE_DIR) !== false
+    if (strpos($MODULE_DIR, '/recipe-') !== false
+        || strpos($MODULE_DIR, '/silverstripe-installer') !== false
     ) {
         return true;
     }
     return false;
 }
 
+/**
+ * Determine if the module being processed is something installed on a website e.g. gha-*
+ *
+ * Example usage:
+ * module_is_not_for_cms()
+ */
+function module_is_not_for_cms()
```

Comment:
```suggestion
 * Determine if the repository represents a silverstripe module.
 * For the purposes of this module, recipes and themes are modules.
 *
 * Example usage:
 * is_module()
 */
function is_module()
```
You'll need to adjust the logic accordingly. Probably best would be to check if there's a composer.json file. If no composer, it's not a module. Then you can check the `type` key in composer.json for silverstripe-vendormodule, silverstripe-module (can skip this one if you like, it was the ss3 type), silverstripe-theme, and silverstripe-recipe

You could call it `is_not_module()` but then you end up with `if (!is_not_module())`, and double negatives are bad.

Reply:
Updated

Reply:
Please create a card to update logic as per above

Reply:
https://github.com/silverstripe/module-standardiser/issues/4